packages/website/vite.config.ts

```diff
@@ -1,23 +1,23 @@
-import url from "node:url";
-import path from "node:path";
+import url from 'node:url';
+import path from 'node:path';
 
-import { defineConfig } from "waku/config";
+import { defineConfig } from 'waku/config';
+import { loadEnv } from 'vite';
 
-const modulesRoot = path.join(
-  path.dirname(url.fileURLToPath(import.meta.url)),
-  "src",
-);
+const modulesRoot = path.join(path.dirname(url.fileURLToPath(import.meta.url)), 'src');
 
-export default defineConfig({
-  root: path.dirname(url.fileURLToPath(import.meta.url)),
-  build: {
-    rollupOptions: {
-      output: {
-        // FIXME this doesn't seem to provide nice output.
-        // TODO we should use `input` instead.
-        preserveModules: true,
-        preserveModulesRoot: modulesRoot,
+export default ({ mode }) => {
+  process.env = { ...process.env, ...loadEnv(mode, process.cwd()) };
+
+  return defineConfig({
+    root: path.dirname(url.fileURLToPath(import.meta.url)),
+    build: {
+      rollupOptions: {
+        output: {
+          preserveModules: true,
+          preserveModulesRoot: modulesRoot,
+        },
       },
     },
-  },
-});
+  });
+};
```

Comment:
We can probably remove this file, because now the website doesn't use waku/router.